core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -145,6 +145,7 @@
   <script src="/templates/dev/head/services/SiteAnalyticsService.js"></script>
 
   <script src="/templates/dev/head/pages/state_editor/StateEditorService.js"></script>
+  <script src="/templates/dev/head/pages/state_editor/StatePropertyServices.js"></script>
```

Comment:
This was required as Collection tab uses ResponsesServices which uses EditiorStatesServices.